apps/ledger-live-desktop/src/renderer/families/evm/StakeFlowModal/index.tsx

```diff
@@ -26,28 +175,169 @@ const StakingModal = ({ account, hasCheckbox, singleProviderRedirectMode, source
+              {formattedProviders.length === 0 && (
+                <Flex justifyContent="center" py={20} width="100%">
+                  <EthStakeIllustration size={140} />
+                </Flex>
+              )}
+              <Box width="100%" height="100%" display="grid">
+                {OPTION_VALUES.map(x => (
+                  <Text
+                    style={{
+                      justifySelf: "center",
+                      visibility: x === selected ? "visible" : "hidden",
+                      gridArea: "1 / 1 / 2 / 2",
+                    }}
+                    key={x}
+                    textAlign="center"
+                    color="neutral.c70"
+                    fontSize={14}
+                    maxWidth={360}
+                  >
+                    {t(`ethereum.stake.category.${x}.description`)}
+                  </Text>
+                ))}
+              </Box>
+            </Flex>
+            <Box
+              alignItems="center"
+              flexShrink={0}
+              justifyContent="center"
+              position="absolute"
+              right={1}
+              top={3}
+            >
+              <IconButton
+                title={t("common.close")}
+                data-testid="modal-close-button"
+                onClick={onClose}
+              >
+                <Icons.Close size="M" />
+              </IconButton>
+            </Box>
+          </Header>
+          <ScrollableContainer
+            ref={scrollableContainerRef}
+            backgroundColor={colors.background.main}
+            flex={1}
+            overflow="auto"
+            justifyContent="center"
+            paddingX={3}
+            paddingY={0}
+            isScrollable={isScrollable}
+            width="100%"
+          >
+            <Box height={SHADOW_HEIGHT * 0.8} />
+            <TrackPage category="ETH Stake Modal" name="Main Modal" />
+            <EthStakingModalBody
+              onClose={onClose}
+              account={account}
+              source={source}
+              providers={formattedProviders}
+            />
+            <Box height={SHADOW_HEIGHT * 0.8} />
+          </ScrollableContainer>
+          <Footer
+            bottom={0}
+            px={4}
+            pb={3}
+            pt={0}
+            width="100%"
+            position="relative"
+            isScrollable={isScrollable}
+          >
+            <Flex
+              alignItems="center"
+              backgroundColor={colors.primary.c20}
+              borderRadius={2}
+              columnGap={3}
+              p={2}
+              style={{
+                cursor: "pointer",
+              }}
+              width="100%"
+              onClick={() =>
+                openURL(
+                  selected === "restaking"
+                    ? "https://www.ledger.com/academy/what-is-ethereum-restaking"
+                    : "https://www.ledger.com/academy/ethereum-staking-how-to-stake-eth",
+                )
```

Comment:
Maybe add the urls here : apps/ledger-live-desktop/src/config/urls.ts so we centralize every urls